crates/oxc_linter/src/utils/config.rs

```diff
@@ -0,0 +1,45 @@
+use serde_json::Value;
+
+/**
+ * parse config for multiple style
+ *
+ * {
+ *  "rules": {
+ *    "jsx-a11y/rule-name": 2
+ *  }
+ * }
+ * or
+ * {
+ *  "rules": {
+ *     "jsx-a11y/rule-name": {
+ *         "ignoreNonDOM": true
+ *       }
+ *    }
+ * }
+ * or
+ *
+ * {
+ *  "rules": {
+ *     "jsx-a11y/rule-name": [{
+ *         "ignoreNonDOM": true
+ *       }
+ *    }]
+ * }
+ */
```

Comment:
i don't think we should have this file - instead do it direcly inside `from_configuration`

I also think that we always do the following:

if the config looks like 
```json
{ 
    "<RULE NAME>": [2, { "config": {} }],
}
```

we just pass in `[{config}]` to the `from_configuration` call, ignoreing the first element.

based on the eslint docs, if a rule configuration is passed, it _has_ to look like `<RULE NAME>: [<SEVERITY>, config]`